Project.toml

```diff
@@ -18,7 +19,8 @@ ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Quaternions = "94ee1d12-ae83-5a48-8b1c-48b8ff168ae0"
```

Comment:
You shouldn't need it here as well as above (same for `Random`)

Reply:
Thanks, I also removed `Quaternions` and `Random` from `targets/test`.